assets/src/dashboard/components/typeahead-options/utils/getTypeaheadOptions.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import memoize from 'lodash/memoize'; // only lodash/memoize is added to package dependencies to avoid code bloat
```

Comment:
I think we earlier agreed not to use lodash. Is it really useful here? So far a few items we tried to use were trivial.

/to @swissspidy to approve.

Reply:
Agreed. We don't currently use lodash anywhere, even though it would even be readily available in WP context.

Could we turn this into a hook and use `useMemo`?

Reply:
@mariano-formidable and I were discussing this if you want to reference our conversation. Am happy to sub out for useMemo but it will need to be tied to the actual data to properly update _or_ the typeahead will need to be configured to either tend to filtering it's own data or not. I can add a new prop that flips how the data's filtered (either by the typeahead or by the parent with a live connection). 

good to know about the use of lodash. I believe that this is a conversation that was had before formidable arrived on it.

Reply:
Yeah that was a while ago when we had this discussion, so no worries!

It's also why you had to disable the `import/no-extraneous-dependencies` rule, because `lodash` isn't listed as a dependency in `package.json` for that reason 🙂 

If using `useMemo` is too cumbersome or not ideal, we can revisit this decision after alpha or alternatively use a dedicated package (e.g. [memize](https://www.npmjs.com/package/memize)) for this.

Reply:
@swissspidy no worries, already swapped it out and pushed up a version that just uses a hook.